lib/mixlib/install/options.rb

```diff
@@ -71,8 +79,7 @@ def validate_channels
       end
 
       def validate_unstable_version
-        if ARTIFACTORY_CHANNELS.include?(channel) &&
-            product_version !~ /^\d+.\d+.\d+\+[0-9]{14}$/
+        if for_artifactory? && product_version !~ /^\d+.\d+.\d+\+[0-9]{14}$/
```

Comment:
So not sure we can use the version number pattern to determine the channel. Even non-integration build versions will flow through unstable. We are also moving to a place where timestamps might be dropped completely. See this RFC for more details:
https://github.com/chef/chef-rfc/blob/master/rfc047-release-process.md